File: 0242-valid-anagram/0242-valid-anagram.py
class Solution(object):
    def isAnagram(self, s, t):
        
        # Convert strings to lists of characters
        list_s = list(s)
        list_t = list(t)

        # Sort the lists
        list_s.sort()
        list_t.sort()

        # Check if the sorted lists have the same content
        return list_s == list_t
        

File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {

        // Convert strings to character arrays
        char[] array_s = s.toCharArray();
        char[] array_t = t.toCharArray();

        // Sort the arrays
        Arrays.sort(array_s);
        Arrays.sort(array_t);

        // Check if the sorted arrays have the same content
        return Arrays.equals(array_s, array_t);
    }
}